buildSrc/src/main/kotlin/Dependency.kt

```diff
@@ -3,10 +3,11 @@
 object Dependency {
 
     object Kotlin {
-        const val stdlib = "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+        const val version = "1.8.0"
```

Comment:
Kotlin 1.8

Reply:
Já vím. Já jsem chtěl mít všude tu 1.7.20, ale když to nastavím pro tu "org.jetbrains.kotlin:kotlin-bom:$version" knihovnu, tak to nejde zbuildit...Mmt, napíšu ti zprávu